environment.yml

```diff
@@ -5,57 +5,66 @@ channels:
   - plotly
   - defaults
 dependencies:
+  # mdanalysis dependencies
   - cython
-  - distributed==2021.1.0
-  - dask==2021.1.0
-  - python-graphviz
-  - pytest
-  - decorator
-  - docutils
-  - graphviz
-  - ipywidgets<8.0.0
-  - ipython
-  - ipygany
-  - jedi
-  - jinja2
-  - jupyter_contrib_nbextensions
   - matplotlib
-  - nbclient
-  - nbconvert
-  - nbformat
-  - nbsphinx
   - networkx
-  - nglview
-  - notebook
   - numpy
+  - openmm>=7.0.0
   - packaging
   - parmed
-  - parso
-  - pip
-  - plotly
-  - pybtex
-  - python=3.9
-  - pyvista
-  - scikit-image
   - scikit-learn>=1.1.2
+  - threadpoolctl
+  # building documentation
+  - docutils
+  - pandoc
+  - pybtex
   - sphinx
   - sphinxcontrib-bibtex
-  - threadpoolctl
+  # used by custom_parallel_analysis.ipynb
+  - graphviz
   - tabulate
-  - widgetsnbextension
-  - ipycytoscape>=1.3.0
+  - python-graphviz
+  # used in multiple notebooks
+  - jupyter_contrib_nbextensions
+  - nbconvert
+  - nbformat
+  - nglview
+  - notebook<7
   - seaborn>=0.12.1
-  - openmm>=7.0.0
-  - pandoc
+  # used in pairwise_rmsd.ipynb
+  - plotly
+  # used in custom_parallel_analysis.ipynb
+  - distributed==2021.1.0
+  - dask==2021.1.0
+  # used in density_analysis.ipynb
+  - ipygany
+  # used in density_analysis.ipynb
+  - pyvista
+  # used in density_analysis.ipynb
+  - scikit-image
+  # used in aligning_trajectory.ipynb
   - moviepy
-  - ipyvtklink
+  # used in conf.py
```

Comment:
I mostly think of these as sphinx "extensions" -- we use the `ipython` directive in some docs, `nbsphinx` to render notebooks, etc. If they're imported/used in `conf.py` it would generally be in their role as extensions.

Reply:
Thanks, yeah, easy to change

Reply:
Thanks @jandom -- could you please move these up to the #extensions then?

Reply:
Just this to go -- nbsphinx, ipywidgets and the like are used to render notebooks, and IIRC ipython to use the `ipython` sphinx directive. If they're imported in conf.py it's to configure settings but, otherwise they fall into the same category as other extensions, e.g. sphinx bibtex.